lib/twitter-ads/creative/cards_fetch.rb

```diff
@@ -0,0 +1,119 @@
+      property :deleted, type: :bool, read_only: true
+      property :duration_in_minutes, read_only: true
+      property :end_time, type: :time, read_only: true
+      property :first_choice, read_only: true
+      property :first_cta, read_only: true
+      property :first_cta_tweet, read_only: true
+      property :first_cta_welcome_message_id, read_only: true
+      property :fouth_choice, read_only: true
+      property :fouth_cta, read_only: true
+      property :fouth_cta_tweet, read_only: true
+      property :fourth_cta_welcome_message_id, read_only: true
+      property :googleplay_app_id, read_only: true
+      property :googleplay_deep_link, read_only: true
+      property :id, read_only: true
+      property :image, read_only: true
+      property :image_display_height, read_only: true
+      property :image_display_width, read_only: true
+      property :ipad_app_id, read_only: true
+      property :ipad_deep_link, read_only: true
+      property :iphone_app_id, read_only: true
+      property :iphone_deep_link, read_only: true
+      property :name, read_only: true
+      property :preview_url, read_only: true
+      property :recipient_user_id, read_only: true
+      property :second_choice, read_only: true
+      property :second_cta, read_only: true
+      property :second_cta_tweet, read_only: true
+      property :second_cta_welcome_message_id, read_only: true
+      property :start_time, type: :time, read_only: true
+      property :thank_you_text, read_only: true
+      property :thank_you_url, read_only: true
+      property :third_choice, read_only: true
+      property :third_cta, read_only: true
+      property :third_cta_tweet, read_only: true
+      property :third_cta_welcome_message_id, read_only: true
+      property :title, read_only: true
+      property :updated_at, type: :time, read_only: true
+      property :video_content_id, read_only: true
+      property :video_height, read_only: true
+      property :video_hls_url, read_only: true
+      property :video_owner_id, read_only: true
+      property :video_poster_height, read_only: true
+      property :video_poster_url, read_only: true
+      property :video_poster_width, read_only: true
+      property :video_width, read_only: true
+      property :video_url, read_only: true
+      property :website_dest_url, read_only: true
+      property :website_display_url, read_only: true
+      property :website_shortened_url, read_only: true
+      property :website_title, read_only: true
+      property :website_url, read_only: true
+      property :wide_app_image, read_only: true
+      property :id, read_only: true
+
+      FETCH_URI = "/#{TwitterAds::API_VERSION}/" +
+                  'accounts/%{account_id}/cards'.freeze # @api private
+      FETCH_ID  = "/#{TwitterAds::API_VERSION}/" +
+                  'accounts/%{account_id}/cards/all/%{id}'.freeze # @api private
+
+      def all(*)
+        raise ArgumentError.new(
```

Comment:
What do you think about raising `NoMethodError` instead? With some string like:

>"NoMethodError: undefined method `all' for CardsFetch"

[Source](http://ruby-doc.org/core-1.9.3/NoMethodError.html)